src/uct/ib/base/ib_md.c

```diff
@@ -1221,6 +1223,22 @@ static ucs_status_t uct_ib_query_md_resources(uct_component_t *component,
         num_resources++;
     }
 
+    if (num_resources != 0) {
+        status = ucs_sys_get_memlock_rlimit(&memlock_limit);
+        if ((status == UCS_OK) && (memlock_limit <= (500 * UCS_MBYTE))) {
```

Comment:
maybe define this minimal size as a configuration parameter (in IB memory domain). default is "512MB"

Reply:
I think the only thing that the user may want to do is just to disable this check and always enable RDMA transport. IMHO the most right semantic for such a control is bool env variable. WDYT?